api/docs/tough-cookie.permutepath.md

```diff
@@ -0,0 +1,61 @@
+<!-- Do not edit this file. It is automatically generated by API Documenter. -->
+
+[Home](./index.md) &gt; [tough-cookie](./tough-cookie.md) &gt; [permutePath](./tough-cookie.permutepath.md)
+
+## permutePath() function
+
+Generates the permutation of all possible values that [pathMatch()](./tough-cookie.pathmatch.md) the `path` parameter. The array is in longest-to-shortest order. Useful when building custom [Store](./tough-cookie.store.md) implementations.
+
+**Signature:**
+
+```typescript
+export declare function permutePath(path: string): string[];
+```
+
+## Parameters
+
+<table><thead><tr><th>
+
+Parameter
+
+
+</th><th>
+
+Type
+
+
+</th><th>
+
+Description
+
+
+</th></tr></thead>
+<tbody><tr><td>
+
+path
+
+
+</td><td>
+
+string
+
+
+</td><td>
+
+the path to generate permutations for
+
+
+</td></tr>
+</tbody></table>
+**Returns:**
+
+string\[\]
+
+## Example
+
+
+```
```

Comment:
Examples seem to be missing the `ts` language tag for syntax highlighting. Is there any way to add it?

Reply:
Yes, it looks like adding `ts` to the block is respected in the output. 

I believe I missed this for several of the PRs. Once these are all merged into the `v5-docs` branch I'll need to do a final bit of cleanup (e.g.; removing the doc `.gitignore`, verifying there are no warnings about missing types from our public api, etc.) so I'll add the following to the list:
- annotate examples with `ts` for syntax highlighting